passenger/templates/default/nginx.conf.erb

```diff
@@ -17,6 +17,9 @@ http {
   passenger_ruby <%= @ruby_path %>;
   passenger_show_version_in_header off;
   passenger_user <%= @passenger_user %>;
+<% @passenger[:pre_start].each do |url| %>
```

Comment:
Should we drop this if we're not using it?

Reply:
My thought was that it can't hurt to keep support for the config. There could be a use for it in the future. That said, it's easy to remove, what do you think?

Reply:
No strong opinion here